<ui version="4.0" >
 <class>ReplaceDialog</class>
 <widget class="QDialog" name="ReplaceDialog" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>518</width>
    <height>262</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Find or Replace a text</string>
  </property>
  <layout class="QVBoxLayout" >
   <property name="margin" >
    <number>9</number>
   </property>
   <property name="spacing" >
    <number>6</number>
   </property>
   <item>
    <layout class="QGridLayout" >
     <property name="margin" >
      <number>0</number>
     </property>
     <property name="spacing" >
      <number>6</number>
     </property>
     <item row="1" column="0" >
      <widget class="QCheckBox" name="m_replaceCheckBox" >
       <property name="text" >
        <string>&amp;Replace by :</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1" >
      <widget class="QComboBox" name="m_comboFind" >
       <property name="editable" >
        <bool>true</bool>
       </property>
       <property name="currentIndex" >
        <number>-1</number>
       </property>
       <property name="insertPolicy" >
        <enum>QComboBox::InsertAlphabetically</enum>
       </property>
      </widget>
     </item>
     <item row="0" column="0" >
      <widget class="QLabel" name="label" >
       <property name="sizePolicy" >
        <sizepolicy>
         <hsizetype>1</hsizetype>
         <vsizetype>5</vsizetype>
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text" >
        <string>Find &amp;what :</string>
       </property>
       <property name="buddy" >
        <cstring>m_comboFind</cstring>
       </property>
      </widget>
     </item>
     <item row="1" column="1" >
      <widget class="QComboBox" name="m_comboReplace" >
       <property name="enabled" >
        <bool>false</bool>
       </property>
       <property name="editable" >
        <bool>true</bool>
       </property>
       <property name="insertPolicy" >
        <enum>QComboBox::InsertAlphabetically</enum>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QGridLayout" >
     <property name="margin" >
      <number>0</number>
     </property>
     <property name="spacing" >
      <number>6</number>
     </property>
     <item row="1" column="1" >
      <spacer>
       <property name="orientation" >
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" >
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="0" column="1" >
      <widget class="QGroupBox" name="m_directionGroupBox" >
       <property name="title" >
        <string>Direction</string>
       </property>
       <layout class="QVBoxLayout" >
        <property name="margin" >
         <number>9</number>
        </property>
        <property name="spacing" >
         <number>6</number>
        </property>
        <item>
         <widget class="QRadioButton" name="m_UpRadioButton" >
          <property name="text" >
           <string>Go up</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="m_DownRadioButton" >
          <property name="text" >
           <string>Go down</string>
          </property>
          <property name="checked" >
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item row="1" column="2" >
      <spacer>
       <property name="orientation" >
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" >
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="0" column="2" >
      <widget class="QGroupBox" name="m_selectionGroupBox" >
       <property name="title" >
        <string>Extend</string>
       </property>
       <layout class="QVBoxLayout" >
        <property name="margin" >
         <number>9</number>
        </property>
        <property name="spacing" >
         <number>6</number>
        </property>
        <item>
         <widget class="QRadioButton" name="m_searchAllRadioButton" >
          <property name="text" >
           <string>All</string>
          </property>
          <property name="checked" >
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="m_searchSelectionRadioButton" >
          <property name="text" >
           <string>Selection</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item rowspan="2" row="0" column="0" >
      <widget class="QGroupBox" name="groupBox" >
       <property name="title" >
        <string>Options</string>
       </property>
       <layout class="QVBoxLayout" >
        <property name="margin" >
         <number>9</number>
        </property>
        <property name="spacing" >
         <number>6</number>
        </property>
        <item>
         <widget class="QCheckBox" name="m_caseCheckBox" >
          <property name="text" >
           <string>Match c&amp;ase</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="m_fromStartCheckBox" >
          <property name="text" >
           <string>Search from &amp;start</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="m_wholeWordsCheckBox" >
          <property name="text" >
           <string>&amp;Whole words</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="m_regularExpressionCheckBox" >
          <property name="text" >
           <string>&amp;Use regular expression</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" >
     <property name="margin" >
      <number>0</number>
     </property>
     <property name="spacing" >
      <number>6</number>
     </property>
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" >
        <size>
         <width>131</width>
         <height>31</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="m_findButton" >
       <property name="text" >
        <string>Find</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="m_closeButton" >
       <property name="text" >
        <string>Close</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>m_comboFind</tabstop>
  <tabstop>m_replaceCheckBox</tabstop>
  <tabstop>m_comboReplace</tabstop>
  <tabstop>m_caseCheckBox</tabstop>
  <tabstop>m_fromStartCheckBox</tabstop>
  <tabstop>m_wholeWordsCheckBox</tabstop>
  <tabstop>m_regularExpressionCheckBox</tabstop>
  <tabstop>m_UpRadioButton</tabstop>
  <tabstop>m_DownRadioButton</tabstop>
  <tabstop>m_searchAllRadioButton</tabstop>
  <tabstop>m_searchSelectionRadioButton</tabstop>
  <tabstop>m_findButton</tabstop>
  <tabstop>m_closeButton</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>m_findButton</sender>
   <signal>clicked()</signal>
   <receiver>ReplaceDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>444</x>
     <y>287</y>
    </hint>
    <hint type="destinationlabel" >
     <x>96</x>
     <y>254</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_closeButton</sender>
   <signal>clicked()</signal>
   <receiver>ReplaceDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>525</x>
     <y>287</y>
    </hint>
    <hint type="destinationlabel" >
     <x>179</x>
     <y>282</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_replaceCheckBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>m_comboReplace</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>137</x>
     <y>48</y>
    </hint>
    <hint type="destinationlabel" >
     <x>398</x>
     <y>48</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_fromStartCheckBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>m_DownRadioButton</receiver>
   <slot>setChecked(bool)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>114</x>
     <y>154</y>
    </hint>
    <hint type="destinationlabel" >
     <x>291</x>
     <y>136</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_fromStartCheckBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>m_searchAllRadioButton</receiver>
   <slot>setChecked(bool)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>150</x>
     <y>152</y>
    </hint>
    <hint type="destinationlabel" >
     <x>406</x>
     <y>102</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_fromStartCheckBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>m_UpRadioButton</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>193</x>
     <y>147</y>
    </hint>
    <hint type="destinationlabel" >
     <x>291</x>
     <y>107</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_fromStartCheckBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>m_searchSelectionRadioButton</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>197</x>
     <y>157</y>
    </hint>
    <hint type="destinationlabel" >
     <x>421</x>
     <y>143</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_fromStartCheckBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>m_regularExpressionCheckBox</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>75</x>
     <y>151</y>
    </hint>
    <hint type="destinationlabel" >
     <x>74</x>
     <y>210</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
